diesel_cli/src/main.rs

```diff
@@ -225,7 +225,24 @@ fn create_migrations_dir(matches: &ArgMatches) -> DatabaseResult<PathBuf> {
             .join("migrations"),
     };
 
-    if !dir.exists() {
+    if dir.exists() {
+        // This is a cleanup code for migrating from an
+        // older version of diesel_cli that set a `.gitkeep` instead of a `.keep` file.
+        // TODO: remove this after a few releases
+        if let Some(dir_entry) = fs::read_dir(&dir)
```

Comment:
As this is purly there because we do want to do some cleanup here we should never fail here. So do not use any code that may panic (so no `unwrap`'s) in the following code.

Reply:
I've just removed `unwrap`!